packages/schema/src/utils.ts

```diff
@@ -151,3 +151,7 @@ export function omitKeysFromObject(
 export function objectKeyEncode(key: string): string {
   return key.indexOf(' ') !== -1 ? literalToString(key) : key;
 }
+
+export function isNullOrMissing(value: unknown): value is null | undefined {
+  return value === null || value === undefined;
```

Comment:
its generally considered a good practice to use the `typeof `operator check the type undefined i.e `return typeof value === 'undefined' || value === null;`

Reply:
Added suggestion